tests/data/Calculation/MathTrig/SQRT.php

```diff
@@ -1,10 +1,16 @@
 <?php
 
 return [
-    ['#VALUE!'], // exception not enough args
-    ['#VALUE!', '"ABC"'],
+    ['exception'], // exception not enough args
```

Comment:
Just as a future reference; a useful trick with unit test dataproviders is to use an associative key for each test case, e.g.
```
'exception not enough args' => ['exception'],
'Non-numeric first argument' => ['#VALUE!', '"ABC"'],
```
It makes things a lot easier to see which test cases have failed when running tests, because PHPUnit doesn't simply report it as failing test case <#n>, but as failing the `exception not enough args` test